### PR TITLE
Update NU5 mainnet activation height.

### PR DESCRIPTION
**Pull request by nuttycom**

### CODE REVIEW

**Review by steven-ecc — APPROVED** [review]
utACK

File: zcash_primitives/src/consensus.rs [hunk 1]
```diff
@@ -659,9 +659,13 @@ mod tests {
...
             BranchId::Canopy,
         );
+        assert_eq!(
+            BranchId::for_height(&MAIN_NETWORK, BlockHeight(1_687_104)),
+            BranchId::Nu5,
+        );
         assert_eq!(
             BranchId::for_height(&MAIN_NETWORK, BlockHeight(5_000_000)),
-            BranchId::Canopy,
+            BranchId::Nu5,
```
Comment (steven-ecc) [inline zcash_primitives/src/consensus.rs:668]: Should this be "NU5"?

Comment (steven-ecc) [reply zcash_primitives/src/consensus.rs:668]: Nm, looks like the constant is mixed case.

Comment (nuttycom) [reply zcash_primitives/src/consensus.rs:668]: Yes; Rust convention is to use CamelCase even for acronyms.

**Review by daira — APPROVED** [review]
utACK

**Pull request merged**